tests/scalers_go/postgresql/postgresql_test.go

```diff
@@ -185,6 +186,38 @@ spec:
   type: ClusterIP
 `
 
+	lowLevelRecordsJobTemplate = `apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: postgresql-insert-low-level-job
+  name: postgresql-insert-low-level-job
+  namespace: {{.TestNamespace}}
+spec:
+  template:
+    metadata:
+      labels:
+        app: postgresql-insert-low-level-job
+    spec:
+      containers:
+      - image: ghcr.io/kedacore/tests-postgresql
+        imagePullPolicy: Always
+        name: postgresql-processor-test
+        command:
+          - /app
+          - insert
+        env:
+          - name: TASK_INSTANCES_COUNT
+            value: "20"
```

Comment:
I'd say that you need to insert less than `activationTargetQueryValue`. Otherwise, the trigger will be activated

Reply:
Query is calculated this way: query: "SELECT CEIL(COUNT(*) / 5) FROM task_instance WHERE state='running' OR state='queued'"

Reply:
yes, all new tasks are inserted with status `queued`, so you are inserting 20 and your activation threshold is 5, it'll be active 
https://github.com/kedacore/test-tools/blob/5b5b8ee3ed5946c552f8b93ed1b5b1bc1efcd4ea/e2e/images/postgresql/cmd/main.go#L51